src/read.c

```diff
@@ -2506,10 +2506,10 @@ static void RecreateStackNams(ReaderState * rs, Obj context)
 **  will be set to 1 if the command was followed by a double semicolon, else
 **  it is set to 0. If 'dualSemicolon' is zero then it is ignored.
 */
-ExecStatus ReadEvalCommand(Obj            context,
-                           TypInputFile * input,
-                           Obj *          evalResult,
-                           BOOL *         dualSemicolon)
+UInt ReadEvalCommand(Obj            context,
```

Comment:
This change was necessary because in `read.h` this function is declared as returning `UInt`, which is now a different type than `ExecStatus`. I have another patch ready which builds on this one and changes it to use `ExecStatus`, but that requires moving the `typedef ExecStatus` to `common.h` and I'd rather not mix these changes unnecessarily.